docs: Minor formatting & punctuation tweaks to native-image.md (#17013)
electron/electron: docs/api/native-image.md

  * `width` Integer (optional) - Defaults to the image's width.
  * `height` Integer (optional) - Defaults to the image's height.
  * `quality` String (optional) - The desired quality of the resize image.

>     Possible values are `good`, `better`, or `best`. The default is `best`.

Reviewer: We can make this `String` be typed as an enum if we write "Can be X, Y or Z"

```suggestion
    Can be `good`, `better` or `best`. The default is `best`.
```

Reviewer 2: @Zearin i'll approve once the above comment has been addressed ☝️
